### PR TITLE
Switch to most capped planet when only easy

### PR DESCRIPTION
**Pull request by ALEEF02**
I made it so when there are only easy tiles, it switches to the planet with the most completion %, so we get another planet faster.

### CODE REVIEW

File: cheat.php [hunk 1]
```diff
@@ -557,12 +557,23 @@ function GetBestPlanetAndZone( &$ZonePaces, $WaitTime )
...
-	usort( $Planets, function( $a, $b )
+	
+	if( $Planet[ 'high_zones' ] == 0 && $Planet[ 'medium_zones' ] == 0 )
 	{
-		return $b[ 'sort_key' ] - $a[ 'sort_key' ];
-	} );
-
+		usort( $Planets, function( $a, $b )
+		{
+			return $a[ 'state' ][ 'capture_progress' ] < $b[ 'state' ][ 'capture_progress' ];
```
Comment (Kowiz) [inline cheat.php]: return type of comparison function is supposed to be an int

Comment (ALEEF02) [reply cheat.php]: Why does it matter? It still works, I've tested it.

Comment (ALEEF02) [reply cheat.php]: There are no problems with it being a float

Comment (Kowiz) [reply cheat.php]: It's not a float, you're returning a boolean true/false value, which php is parsing as a 1/0.
Standards use negative number, zero, positive number.  By returning a boolean value, you are losing the negative number.
Zero is used to denote that the values are equal.
Basically what your code is doing is: if A is greater than B, return a 0 to tell the sort method that they're equal.

Your code might be working, but it's not working for the right reasons.

**Comment by iAlcatrazZ** [comment]
hi. is there a way that i can change the timezone??? it's currently lock to gmt, i'm sorry i really dont know where to ask

**Comment by ALEEF02** [comment]
@iAlcatrazZ Go to the `php.ini` file in your PHP folder. On line 939, remove the `;`. Next, go to the link on line 938 and select your timezone. Then paste it after the `=`. It should look something like `date.timezone = "America/Los_Angeles"` When done. Now save the file and restart the script and you should be good.

**Comment by ALEEF02** [comment]
Wasting my time...

**Pull request closed without merging**